runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordGameEventType.java

```diff
@@ -185,4 +264,15 @@ public static DiscordGameEventType fromSkill(final Skill skill)
 			default: return null;
 		}
 	}
+
+	public static  DiscordGameEventType fromLocation(final int regionId)
+	{
+		Optional<DiscordGameEventType>optional = Arrays.stream(DiscordGameEventType.values()).filter(d -> d.regionIds.contains(regionId)).findAny();
```

Comment:
Dat spacing and line length

Reply:
Also, I'm pretty sure Adam won't like this.
Try iterating through the values, return what matches your filter, or return null.

I think the line count will be about the same for this too.